src/Workspaces/CoreTest/Renamer/VisualBasicRenamerTests.cs

```diff
@@ -28,30 +28,41 @@ public Task VisualBasic_RenameDocument_NoRenameType()
         [Fact]
         public Task VisualBasic_RenameDocument_RenameType()
         => TestRenameDocument(
+            MakeSingleDocumentWithInfoArray(
 @"Class OriginalName
 End Class",
+                name: "OriginalName.vb"),
+            MakeSingleDocumentWithInfoArray(
 @"Class NewDocumentName
 End Class",
-            documentName: "OriginalName.vb",
-            newDocumentName: "NewDocumentName.vb");
+                name: "NewDocumentName.vb"),
+            RenameHelpers.MakeSymbolPairs("OriginalName", "NewDocumentName"));
 
         [Fact]
         public Task VisualBasic_RenameDocument_RenameInterface()
         => TestRenameDocument(
+            MakeSingleDocumentWithInfoArray(
 @"Interface IInterface
 End Interface",
+                name: "IInterface.vb"),
+            MakeSingleDocumentWithInfoArray(
 @"Interface IInterface2
 End Interface",
-            documentName: "IInterface.vb",
-            newDocumentName: "IInterface2.vb");
+                name: "IInterface2.vb"),
+            RenameHelpers.MakeSymbolPairs("IInterface", "IInterface2"));
```

Comment:
Don't forget you can write `New Dictionary(Of String) From{ { "IInterface", "IInterface2" } }"  and use collection initializers. On the C# side you might even be able to use target-typed new and omit all sorts of stuff.

Reply:
I've had comments in the past that it's less readable, so I've been doing this style 🤷‍♀️